classes/Application/Speakers.php

```diff
@@ -15,16 +15,16 @@
 class Speakers
 {
     /** @var CallForProposal */
-    protected $callForProposal;
+    private $callForProposal;
```

Comment:
All of these pose potential BC breaks if any of the users of `opencfp/opencfp` extend any of these classes (which I don't know, of course).

Otherwise, totally 👍!